src/views/forms/UnstakeForm/index.tsx

```diff
@@ -121,7 +124,14 @@ export const UnstakeForm = ({ onDone, className }: UnstakeFormProps) => {
     }
     try {
       setIsLoading(true);
-      await undelegate(amounts);
+      const tx = await undelegate(amounts);
+      const txHash = hashFromTx(tx.hash);
+
+      track(AnalyticsEvent.UnstakeTransaction, {
+        txHash,
+        amount: totalAmount,
+        validatorAddresses: Object.keys(amounts),
```

Comment:
would it be worth also incorporating whether the transaction was successful or failed? (i.e. by tracking after onDone and in the catchError?)

Reply:
Should we track on "click"? This is just tracking when it's done right since we get the txn hash